lib/rex/proto/dns/cache.rb

```diff
@@ -21,7 +21,7 @@ def initialize
     # Find entries in cache, substituting names for '*' in return
     #
     # @param search [String] Name or address to search for
-    # @param type [String] Record type to search for
+    # @param type [Dnsruby::Types::*] Record type to search for
```

Comment:
Think this should just be `Dnsruby::Types` since that is the class that contains all these subtypes. YARD doesn't understand the `::*` syntax I don't think unless I'm missing something?

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16390/commits/8969e3130257e238f5523ae20b772ceb8257fcfd